src/annotation.js

```diff
@@ -21,14 +21,20 @@ module.exports = function(Chart) {
 		return function(chartInstance, easingDecimal) {
 			var defaultDrawTime = chartInstance.annotation.options.drawTime;
 
-			helpers.elements(chartInstance)
+			var elements = helpers.elements(chartInstance)
 				.filter(function(element) {
 					return drawTime === (element.options.drawTime || defaultDrawTime);
-				})
-				.forEach(function(element) {
-					element.configure();
-					element.transition(easingDecimal).draw();
 				});
+
+			// Draw our box/line first, then the labels, so the labels will be on top.
+			elements.forEach(function(element) {
+				element.configure();
+				element.transition(easingDecimal).draw();
+			});
+			elements.forEach(function(element) {
+				element.configure();
```

Comment:
do we need to call `configure` a second time?

Reply:
🤔 Good question! I don't know?

Reply:
Ok. Can you test if it works without the duplicate calls here? Let's remove if possible

Reply:
Confirmed, `draw()` doesn't mutate the state of `model` as used in `configure`.  Updated in 44a5432f483c3b707b16dc5e5471ff64a0aa0e70